components/AppBar/src/MDCAppBarNavigationController.m

```diff
@@ -126,6 +126,14 @@ - (void)injectAppBarIntoViewController:(UIViewController *)viewController {
   UIScrollView *trackingScrollView =
       [self findFirstInstanceOfUIScrollViewInView:viewControllerView];
 
+  if ([self.delegate respondsToSelector:@selector
```

Comment:
Tip: Looks like we can improve performance slightly by not searching the view hierarchy unless the delegate fails to provide a tracking scroll view.

Reply:
I guess this isn't possible given the way the API should be used. Resolving.